### PR TITLE
Task/ci release binaries

### PR DESCRIPTION
**Pull request by dimacodota**

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Set env
+        run: echo ::set-env name=RELEASE_VERSION::$(echo ${GITHUB_REF:10})
+      - id: package-version
+        uses: martinbeentjes/npm-get-version-action@master
```
Comment (boaz-codota) [inline .github/workflows/release.yml:17]: fork this and use the fork

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+      - name: Tag & Package.json Versions Not Matching
+        if: ${{ format('v{0}', steps.package-version.outputs.current-version) != env.RELEASE_VERSION}}
+        run: exit 1
+      - uses: UziTech/action-setup-atom@v1
```
Comment (boaz-codota) [inline .github/workflows/release.yml:21]: use a fork

File: dl_binaries.sh [hunk 3]
```diff
@@ -22,3 +22,6 @@ do
     curl -sS https://update.tabnine.com/$path > binaries/$path
     chmod +x binaries/$path
 done
+
```
Comment (boaz-codota) [inline dl_binaries.sh:25]: copy the `dl_binaries.sh` from the repo, as I've also added `rm -rf` to the old folder

**Review by boaz-codota — CHANGES_REQUESTED** [review]

File: .github/workflows/release.yml (hunk 1 above)
Comment (dimacodota) [reply .github/workflows/release.yml]: donne

File: .github/workflows/release.yml (hunk 2 above)
Comment (dimacodota) [reply .github/workflows/release.yml]: done

File: dl_binaries.sh (hunk 3 above)
Comment (dimacodota) [reply dl_binaries.sh:27]: done

**Review by boaz-codota — APPROVED** [review]

**Pull request merged**